styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  width: 375px;
+  height: 590px;
+  margin: 0%;
```

Comment:
- [ ] Please don't set a fixed with of `375px` on the body. The mobile version should be responsive until you get to the desktop screen sizes. 👍